test/functional/fixtures/api/es-next/click/pages/index.html

```diff
@@ -6,11 +6,38 @@
 </head>
 <body>
 <div id="btn" style="background: red; width: 20px; height: 20px;">Button</div>
-
+<div id="div" style="background: grey; width: 100px; height: 100px;">Div</div>
+<!--Click button test-->
 <script>
     document.querySelector('#btn').addEventListener('click', function () {
         throw new Error('Button clicked');
     });
 </script>
+<!--Click without offset options test-->
+<script>
+    var expectedOffset = {
+        x: 50,
+        y: 50
+    };
+
+    document.querySelector('#div').addEventListener('click', function (e) {
+        var rect = document.getElementById('div').getBoundingClientRect();
+
+        var actualOffset = {
+            x: e.pageX - rect.left,
+            y: e.pageY - rect.top
+        };
+
+        if (actualOffset.x === expectedOffset.x && actualOffset.y === expectedOffset.y)
```

Comment:
We can use assertions in TestCafe tests instead of error throwing.
Do like this:
```js
// on the page
window.clickButtonOffset = { x: e.pageX - rect.left, y: e.pageY - rect.top };

// in the test
const getClickButtonOffset = Hybrid(() => window.clickButtonOffset);

var clickButtonOffset = getClickButtonOffset();

expect(...);
```